### ISSUE TITLE
Predicted mask image has different dimension than input image

### ISSUE THREAD

**Issue by spatiallysaying** [timojl/clipseg#43]
I want to overlay the generated masks with transparency on the original image. But, the predicted masks seem to be in a different shape

`image.width,image.height`
(1052, 744)
`
torch.sigmoid(preds[i][0]).cpu().numpy().shape`
(352, 352)

**Comment by timojl** [comment]
I recommend to resize the mask to match the image resolution, e.g. by `torch.nn.functional.resize`.
